SofaKernel/modules/SofaBaseMechanics/src/SofaBaseMechanics/DiagonalMass.inl

```diff
@@ -1152,11 +1140,142 @@ void DiagonalMass<DataTypes, MassType>::computeMass()
                 }
             }
             d_totalMass.setValue(total_mass);
-            d_vertexMass.endEdit();
         }
     }
 }
 
+
+template <class DataTypes, class MassType>
+typename DiagonalMass<DataTypes, MassType>::Real DiagonalMass<DataTypes, MassType>::computeVertexMass(const Real& md)
```

Comment:
```suggestion
typename DiagonalMass<DataTypes, MassType>::Real DiagonalMass<DataTypes, MassType>::computeVertexMass(Real density)
```
In this function definition, you named the parameter with a more meaningful name.